components/Others/styles.tsx

```diff
@@ -0,0 +1,88 @@
+import { StyleSheet } from "react-native";
```

Comment:
More descriptive folder names should be used

Reply:
Created 2 different folders and named them accordingly to resolve in most recent commit